code/game/objects/structures/crates_lockers/closets.dm

```diff
@@ -166,7 +166,6 @@
 	opened = TRUE
 	if(!dense_when_open)
 		density = FALSE
-	climb_time *= 0.5 //it's faster to climb onto an open thing
```

Comment:
lame removal

Reply:
yeah i wasnt sure how to fix this but i thought of a way